hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/pom.xml

```diff
@@ -48,6 +48,12 @@
       <groupId>org.apache.hadoop</groupId>
       <artifactId>hadoop-common</artifactId>
       <scope>provided</scope>
+      <exclusions>
```

Comment:
this is excluded here

Reply:
This is temporary. As current SNAPSHOT version of thirdparty is leading to enforce:depcheck failure due to mismatch of resolved version.

Once release version is used we can remove exclusion.

Reply:
ok, so it'll only be in yarn API?

Reply:
Yes.